index.html; Added Switch-Cond. and Across-Loop

yenicelik opened the pull request:
In the If-Else Header: Switch-Conditional was added (l.373-385)
In the For-Loop Header: Across-Loop was added (l.408-419)

Commit:
index.html; Added Switch-Cond and Across-Loop

In the If-Else Header: Switch-Conditional was added (l.373-385)
In the For-Loop Header: Across-Loop was added (l.408-419)

carlfriess commented on index.html:
```diff
@@ -367,6 +367,22 @@
+    <div class="code"><pre><code class="language-eiffel">
+
+inspect var
+   when case1 then 
```
Könntest du evtl. ein etwas spezifischeres Beispiel geben. Also z.B. mit echten Werten. Ist denke ich einfacher zu verstehen.

carlfriess commented on index.html:
```diff
@@ -367,6 +367,22 @@
+
+    <div class="code"><pre><code class="language-eiffel">
+
+inspect var
```
`var` ist sehr missverständlich... Ich würde einen andern Namen nehmen.

Commit:
index.html; Switch and Across loop examplified

index.html; 
Examples added to Switch and Across loops (instead of abstract variables and text)
